src/main/java/com/opendxl/client/DxlClientConfig.java

```diff
@@ -872,7 +872,8 @@ protected synchronized List<Broker> getSortedBrokerList(final Collection<Broker>
             if (!es.awaitTermination(brokerPingTimeout * 5, TimeUnit.MILLISECONDS)) {
                 es.shutdownNow();
                 if (!es.awaitTermination(brokerPingTimeout * 5, TimeUnit.MILLISECONDS)) {
-                    logger.error("Error shutting down getSortedBrokerList executor service thread pool");
+                    logger.info("Error shutting down getSortedBrokerList executor service thread pool"
```

Comment:
Can you change this to "warn" level and message start with "Not able to shutdown" instead of "Error shutting down.."